nimare/workflows/__init__.py

```diff
@@ -1,7 +1,14 @@
 """Common meta-analytic workflows."""
 
 from .ale import ale_sleuth_workflow
-from .cbma import cbma_workflow
+from .base import Workflow
+from .cbma import CBMAWorkflow, PairwiseCBMAWorkflow
 from .macm import macm_workflow
 
-__all__ = ["ale_sleuth_workflow", "cbma_workflow", "macm_workflow"]
+__all__ = [
+    "ale_sleuth_workflow",
+    "Workflow",
```

Comment:
I would remove the  `Workflow` baseclass from `__all__`, this is not a public facing class (e.g., do not want users importing  and trying to use it by mistake.